docs/google-kubernetes-tutorial.md

```diff
@@ -102,42 +102,56 @@ We can watch the operator come up with:
 
 When you see `Running`, `Control + C` and proceed to launch a TiDB cluster!
 
-## Deploy your first TiDB Cluster
+## Deploy your first TiDB cluster
 
 Now with a single command we can bring-up a full TiDB cluster:
 
 	helm install ./charts/tidb-cluster -n tidb --namespace=tidb --set pd.storageClassName=pd-ssd,tikv.storageClassName=pd-ssd
 
-It will take a few minutes to launch.  You can monitor the progress with:
+It will take a few minutes to launch. You can monitor the progress with:
 
 	watch kubectl get pods --namespace tidb -o wide
 
-When you see all pods `Running`, it's time to `Control + C` and proceed forward!
+The TiDB cluster includes 2 TiDB pods, 3 TiKV pods, and 3 PD pods. When you see all pods `Running`, it's time to `Control + C` and proceed forward!
 
-## Connecting to TiDB
+## Connect to TiDB
 
-There can be a small delay between the pod being up and running, and the service being available.  You can watch list services available with:
+There can be a small delay between the pod being up and running, and the service being available. You can watch list services available with:
 
 	watch "kubectl get svc -n tidb"
 
-When you see `demo-tidb` appear, you can `Control + C`.  The service is ready to connect to!
+When you see `demo-tidb` appear, you can `Control + C`. The service is ready to connect to!
 
 You can connect to the clustered service within the Kubernetes cluster:
 
 	kubectl run -n tidb mysql-client --rm -i --tty --image mysql -- mysql -P 4000 -u root -h $(kubectl get svc demo-tidb -n tidb --output json | jq -r '.spec.clusterIP')
 
-You can also establish a tunnel between an individual pod and your Cloud Shell. This is recommended only for debugging purposes since the tunnel will not automatically be transferred if `demo-tidb-0` fails, or your Cloud Shell restarts. To establish a tunnel:
+Congratulations, you are now up and running with a distributed TiDB database compatible with MySQL!
 
-	kubectl -n tidb port-forward demo-tidb-0 4000:4000 &
+If you like, you can check your connection to TiDB:
+
+	select tidb_version();
+
+In addition to connecting to TiDB within the Kubernetes cluster, you can also establish a tunnel between an individual pod and your Cloud Shell. This is recommended only for debugging purposes, because the tunnel will not automatically be transferred if `demo-tidb-0` fails, or your Cloud Shell restarts. To establish a tunnel:
```

Comment:
In addition to connecting to TiDB within the Kubernetes cluster, you can also establish a tunnel between the TiDB service and your Cloud Shell. This is recommended only for debugging purposes, because the tunnel will not automatically be transferred if your Cloud Shell restarts. To establish a tunnel: